### PR TITLE
Update page_07.rst

### PR DESCRIPTION
**Pull request by anujsoftdel**

### CODE REVIEW

**Review by aditisa1 — APPROVED** [review]

File: source/Pages/page_07.rst [hunk 1]
```diff
@@ -613,11 +616,13 @@ Please note following: This configuration file should be created manually with f
...
 
     e.g., If a point to be polled is not present in datapoints config file, then data for that control loop will not be collected.
+    
+    B) Polled data points in “Polled_point” and write operation data points in “write_operation” must be unique.
 
-    B)	If the points being polled are configured as *“realtime”* in datapoints config file, then *“isRTModeForPolledPoints”* should be set to *“true”*. It should be set to *“false”* otherwise.
+    C)	If the points being polled are configured as *“realtime”* in datapoints config file, then *“isRTModeForPolledPoints”* should be set to *“true”*. It should be set to *“false”* otherwise.
 
-    C)	ZMQ-based KPI App can monitor either RT or Non-RT points at a time.
+    D)	ZMQ-based KPI App can monitor either RT or Non-RT points at a time.
```
Comment (nagdeepgk) [inline source/Pages/page_07.rst:624]: Why only ZMQ based KPI App ? Both MQTT based and ZMQ based app should be the case rigth ?
@anujsoftdel : Cant we monitor both RT or Non-RT points  monitored together ?

Comment (bdasamara) [reply source/Pages/page_07.rst:624]: i would like to see the answer to Nagdeep's questions. are they here?

Comment (nagdeepgk) [reply source/Pages/page_07.rst:624]: @shivaji @anujsoftdel : Request to please update.
CC: @bdasamara @vats-in

**Comment by nagdeepgk** [comment]
Closing this PR because, i have submitted this changes in another PR (created by Intel) https://github.com/open-edge-insights/uwc-docs/pull/39 & merged, by incorporating my review comments myself.

**Pull request closed without merging**